charts/kubescape-operator/README.md

```diff
@@ -28,22 +28,12 @@ kubescape     storage-59567854fd-hg8n8                            1/1     Runnin
 
 The scanning results will be available gradually as the scans are completed.
 
-View your configuration scan summaries:
```

Comment:
@matthyx we should revert both changes

Reply:
Was this fully removed from the storage?

Reply:
configurationscansummaries and vulnerabilitysummaries are still here - but they are virtual objects

Reply:
It looks like the storage is not creating them...
```
% kubectl get vulnerabilitymanifestsummaries -A
No resources found
```

Reply:
And I'm running the latest tag.